### DISCUSSION TITLE
Page  data not fetching in Looker Studio with Epilocal Connector

### DISCUSSION THREAD

**Discussion by Eurofins2023** [plausible/analytics#2952] [Self-Hosted Support]
I have tried connecting Plausible to looker studio with Epilocal Connector 

Only page dimension not fetching data with Pageviews, Visitors, Bounce Rate and Visit Duration.

![debug](https://github.com/plausible/analytics/assets/133360425/6e4d6337-a4ee-4e13-bbfb-3854b310bb07)

Kindly assist.

**Comment by Hricha-Shandily** [comment]
We have now built an official Plausible Analytics Looker Studio Connector. All details available in our setup guide: https://plausible.io/docs/looker-studio
